apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -345,22 +345,20 @@ suite('ActivityHandler', function() {
       var sendSpy;
       var findByPromise;
 
-      setup(function() {
+      setup(function(done) {
         sendSpy = this.sinon.spy(window, 'Notification');
         findByPromise = Promise.resolve(
           [ { name: [contactName] } ]
         );
         this.sinon.stub(Contacts, 'findByAddress').returns(findByPromise);
 
-        MockNavigatormozApps.mTriggerLastRequestSuccess();
+        MockNavigatormozApps.mTriggerLastRequestSuccess().then(done, done);
```

Comment:
Definitely!

But how do you feel about improving "getSelf" and make it more generic (since in reality it always returns Promise-based result). Like I attempted to do here, but not finished (I admit I've used too much code, maybe you have better idea) https://github.com/azasypkin/gaia/commit/bc13b20bfd74aecd07cef336071068e6c60019bb#commitcomment-10663074

If you think it's too much changes for this patch, I'm ok as well and your quick fix in mock won't hurt anyone I think.